modules/royalty/index.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const userPattern = new RegExp(/\<@([^\s|\<]+)\>/, 'g');
+const RoyaltyHelper = require('./royalty-helper.js');
```

Comment:
i know this is a huge nit but can you make it a `royalty-util` helpers are generally static and i hate them (in java land)